packages/dev/loaders/src/glTF/2.0/Extensions/KHR_animation_pointer.ts

```diff
@@ -97,26 +97,29 @@ export class KHR_animation_pointer implements IGLTFLoaderExtension {
             throw new Error(`${extensionContext}: Pointer is missing`);
         }
 
+        let obj: { object: any; info: any };
         try {
-            const obj = this._pathToObjectConverter.convert(pointer);
-            if (!obj.info.interpolation) {
-                throw new Error(`${extensionContext}/pointer: Interpolation is missing`);
-            }

```

Comment:
This was inside the catch, so it didn't actually propagate out as the correct error.